neurom/io/hdf5.py

```diff
@@ -184,29 +184,29 @@ def remove_duplicate_points(points, groups):
         '''
 
         group_initial_ids = groups[:, 0]
+        group_len = len(group_initial_ids)
 
         def _find_last_point(group_id):
             ''' Identifies and returns the id of the last point of a group'''
+            return group_initial_ids[group_id + 1] - 1
```

Comment:
I'm not aware of any official 'spec' on ordering of branches.  I suspect that there are places where the expectations is that they match the original neurolucida ascii, and that if you write the file back, they are in the same order.